src/modules/CurrencyModule.h

```diff
@@ -105,6 +107,10 @@ class CurrencyModule : public Module
 		int stealFinePercent { 50 };
 		int stealVictimBonusPercent { 25 };
 		int stealFailedJailTime { 3 };
+
+		int bribeMaxAmount{ 150 };
+		int bribeLeastAmount{ 20 };
+		int bribeSuccessChance{ 69 };
```

Comment:
These should be saved and loaded from the currency config file.

Reply:
Sorry I didn't get your word's.....

Reply:
You should make it so those variables are saved to the JSON file with all the other configuration variables. They’d also be loaded at startup so their values would remain even after the bot shuts down.